makefiles/tools/openocd-adapters/raspi.inc.mk

```diff
@@ -0,0 +1,26 @@
+# Raspberry Pi GPIO
+
+export SWCLK ?= 21
+export SWDIO ?= 20
+export RST ?= 16
+
+ifeq (1, $(shell grep ARMv6 /proc/cpuinfo > /dev/null; echo $?))
+    # raspi2, 3
+    export PERIPH_BASE  = 0x3F000000
+    export SPEED_COEFF  = 146203
+    EXPORT SPEED_OFFSET = 36
```

Comment:
typo in the `EXPORT` here. I'm not sure if these exports are needed though.

Reply:
oops, fixed.